src/h3/_cy/memory.pxd

```diff
@@ -0,0 +1,19 @@
+from .h3lib cimport H3int
+
+cdef class H3MemoryManager:
+    cdef:
+        size_t n
+        H3int* ptr
+
+    cdef H3int[:] create_mv(self)
+
+
+cdef int[:] int_mv(size_t n)
+cdef H3int[:] empty_memory_view()
+
+"""
+todo: read: https://cython.readthedocs.io/en/latest/src/tutorial/pxd_files.html
+
+todo: should i be allocating with python memory functions?
+https://cython.readthedocs.io/en/latest/src/tutorial/memory_allocation.html
```

Comment:
I would expect this is preferred

Reply:
It turns out there are some interesting trade-offs we might consider around acquiring the GIL. Checkout the changes here: https://github.com/uber/h3-py/pull/261/commits/bf5fb8a079b46484d6470f24872e831f769b65e1

Reply:
cc: @kylebarron